Makefile

```diff
@@ -57,8 +61,8 @@ release-build-alpine:
 	cp libwasmvm/target/release/examples/libmuslc.a api/libwasmvm_muslc.a
 	make update-bindings
 	# try running go tests using this lib with muslc
-	docker run --rm -u $(USER_ID):$(USER_GROUP) -v $(shell pwd):/testing -w /testing $(BUILDERS_PREFIX)-alpine go build -tags muslc .
-	docker run --rm -u $(USER_ID):$(USER_GROUP) -v $(shell pwd):/testing -w /testing $(BUILDERS_PREFIX)-alpine go test -tags muslc ./api ./types
+	docker run --rm -u $(USER_ID):$(USER_GROUP) -v $(shell pwd):/mnt/testrun -w /mnt/testrun $(ALPINE_TESTER) go build -tags muslc .
```

Comment:
What about line 60? Also change variable?

Reply:
No, the idea is to allow using separate builder and tester images here. There is no real point the Rust builder image should include a Go dev environment. I used the different values for Rust build and Go test environment when debugging issues documented in #277.